server-plugins/notification-resources/src/index.ts

```diff
@@ -98,6 +98,34 @@ export async function OnBacklinkCreate (tx: Tx, control: TriggerControl): Promis
   return res
 }
 
+/**
+ * @public
+ */
+export async function OnDmCreate (tx: Tx, control: TriggerControl): Promise<Tx[]> {
```

Comment:
I suppose it should be in chunter server plugin.

Reply:
Moved to chunter server plugin, thanks